lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -116,7 +116,7 @@ class TurboGraft.Remote
       enabledInputs.push(input)
     enabledInputs
 
-  onSuccess: (ev) ->
+  onSuccess: (ev) =>
```

Comment:
Figured I'd switch this to a fat arrow too seen as it's always used as a callback